daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala

```diff
@@ -778,6 +800,46 @@ object ScriptF {
     }
   }
 
+  final case class SubmitConcurrentlyData(
+      actAs: OneAnd[Set, Party],
+      readAs: Set[Party],
+      cmdss: List[List[command.ApiCommand]],
+      stackTrace: StackTrace,
+  )
```

Comment:
I dont think we need this indirection here, unless we intend to write submitMustFailConcurrently, and such.
SubmitData only exists because the triplet of sequential submit commands use the same payload, but I don't think we want that triplet for this, especially if its internal only